### PR TITLE
Remove Coveralls

### PR DESCRIPTION
**Pull request by no-reply**
Coveralls reliably reports nonsense from CircleCI, and it's just the worst.

@samvera/hyrax-code-reviewers

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -7,7 +7,6 @@ def ci_build?
 
 require 'simplecov'
 require 'coveralls'
-Coveralls.wear!
 SimpleCov.root(File.expand_path('../..', __FILE__))
 SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([
                                                                  Coveralls::SimpleCov::Formatter,
```
Comment (rotated8) [inline spec/spec_helper.rb]: What is this formatter used for after Coveralls is removed?

Comment (no-reply) [reply spec/spec_helper.rb]: it seemed better to not change the simplecov formatter just to stop reporting failures.

Comment (no-reply) [reply spec/spec_helper.rb]: on the other hand, dropping `Coveralls.wear!` doesn't seem to have fixed the issue at all.

Comment (no-reply) [reply spec/spec_helper.rb]: rebuilding with coveralls dropped entirely.

**Pull request merged**